src/core/operators/test_media_file_hash.py

```diff
@@ -15,6 +15,7 @@ def tearDownClass(cls):
         # delete config files
         pass
 
+    @skip
```

Comment:
hey, just wanted to check why we are skipping this test now

Reply:
because, the media file hash operator now deletes the file, so this function of the test is calling the file from a local folder, we don't want to delete that, thats why the other function where the file is downloaded from a URL and then ran on the operator makes sense now.

Reply:
ok, seems ok for now. May have to come back to this later